### ISSUE TITLE
Map Call Number Location Data for Holding Location into Aeon SubLocation parameter

### ISSUE THREAD

**Issue by kevinreiss** [pulibrary/requests#782]
When passing a request to aeon map the data in the "call_number" field from the key "holdings_1display" in the solr into the "SubLocation" parameter in the Aeon OpenURL sent over to the request form. Example bib IDs:
Alma example:
https://catalog.princeton.edu/catalog/9947201943506421/raw

Old voyager examples:
https://catalog.princeton.edu/catalog/4720194
https://catalog.princeton.edu/catalog/4720194/raw
https://catalog.princeton.edu/catalog/8950331/
https://catalog.princeton.edu/catalog/8950331/raw

**Comment by carolyncole** [comment]
@kevinreiss that data is already on the call number field: https://github.com/pulibrary/requests/blob/master/app/models/concerns/requests/aeon.rb#L103 Did they really want a copy?  right now the location note is being put in: 
https://github.com/pulibrary/requests/blob/master/app/models/concerns/requests/aeon.rb#L121

**Comment by carolyncole** [comment]
From our conversation is slack.  What they really want with a holding record `holdings_1display: "{"4942114":{"location":"Special Collections - Cotsen Children's Library","library":"Special Collections","location_code":"ctsn","call_number":"Euro 20Q 5828","call_number_browse":"5828"}}",` 
is `Euro 20Q` to be put SubLocation

**Comment by kevinreiss** [comment]
Yes, SC considers this portion to be their Holding Location for materials, "SubLocation"

**Comment by carolyncole** [comment]
This looks to be waiting on pulibrary/bibdata#873 there does not seem to be a separate field yet in the holdings information: https://catalog.princeton.edu/catalog/9947201943506421/raw

**Comment by kevinreiss** [comment]
Closing in favor of new OL ticket.
